apps/mdn/mdn-aws/infra/mdn-cdn/mdn_cdn.tf

```diff
@@ -32,3 +33,15 @@ module "mdn-cloudfront-prod" {
 }
 
 
+# Attachments origin
+
+module "mdn-cloudfront-attachments-prod" {
+    source = "./cloudfront_attachments"
```

Comment:
I guess this is why the terraform file has to be in a subdirectory

Reply:
yes, it's just the subdir name for additional terraform files. It helps with Terraform reuse.